Kick the Sheets/Views/Home/Home.swift

```diff
@@ -11,6 +11,25 @@ import SwiftUI
 struct Home: View {
     @StateObject var viewModel: HomeViewModel
 
+    var body: some View {
+        BaseView {
+            ScrollView {
+                VStack(alignment: .center) {
+                    header
+                    calendar
```

Comment:
Testing showed an error with the calendar. The days were offset by 1.

Reply:
NOTE: This is only appears on iOS 17 devices. Will create an issue to address this later.